app/event-attendance/page.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
```

Comment:
I believe that we need to either move the useState Component or add 'useClient' here.

Reply:
i agree, other than that, looks good to me

Reply:
also just getting this error, but other than that looks clean !

Reply:
Oh yes. Just added `use client` as suggested!

Reply:
Closed and opened a new PR!